metadata_inserter.py

```
@@ -135,8 +129,33 @@ def fill_gaps(comic_info: XMLStruc) -> XMLStruc:
+                self.pending = True
         for key in list(data.keys()):
             if key not in MANDATORY_FIELDS and not data[key]:
                 data[key] = "MISSING"
 
         return comic_info.model_copy(update=data)
+
+    def replace_xml(self, xml_bytes: bytes) -> None:
+        temp_path = self.path.with_suffix(".tmp")
+        try:
+            with zipfile.ZipFile(self.path, "r") as zin, \
+            zipfile.ZipFile(temp_path, "w", zipfile.ZIP_DEFLATED) as zout:
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)